src/test/groovy/org/mockito/release/gradle/ReleaseNeededTaskTest.groovy

```diff
@@ -0,0 +1,75 @@
+package org.mockito.release.gradle
+
+import org.gradle.api.GradleException
+import org.gradle.testfixtures.ProjectBuilder
+import org.mockito.release.internal.comparison.PublicationsComparator
+import spock.lang.Specification
+
+class ReleaseNeededTaskTest extends Specification {
```

Comment:
This is a great test. Thank you very much for adding it!!!!!!!

We were long overdue for the unit test for this task.